academic-services/graph/samples-azure-databricks-hindex.md

```diff
@@ -0,0 +1,337 @@
+       FROM AuthorPaperCitation;
+   ```
+
+   Press the **SHIFT + ENTER** keys to run the code in this block. You see following output:
+
+   ```
+   OK
+   ```
+
+1. **Compute h-index for all authors**. Paste the following code in a new cell.
+
+   ```sql
+   %sql
+   -- Generate author hindex
+   CREATE OR REPLACE TEMPORARY VIEW AuthorHIndexTemp
+       AS SELECT
+           AuthorId,
+           SUM(EstimatedCitation) AS TotalEstimatedCitation,
+           MAX(CASE WHEN EstimatedCitation >= Rank THEN Rank ELSE 0 END) AS HIndex
+       FROM AuthorPaperOrderByCitation 
+       GROUP BY AuthorId;
+   ```
+
+   Press the **SHIFT + ENTER** keys to run the code in this block. You see following output:
+
+   ```
+   OK
+   ```
+
+1. **Get author detail information**. Paste the following code in a new cell.
+
+   ```sql
+   %sql
+   -- Get author detail information
+   CREATE OR REPLACE TEMPORARY VIEW AuthorHIndex
+       AS SELECT
+           I.AuthorId,
+           A.DisplayName,
+           AF.DisplayName AS AffiliationDisplayName,
+           A.PaperCount,
+           I.TotalEstimatedCitation,
+           I.HIndex
+       FROM AuthorHIndexTemp AS I
+       INNER JOIN Authors AS A
+           ON A.AuthorId == I.AuthorId
+       LEFT OUTER JOIN Affiliations AS AF
+           ON A.LastKnownAffiliationId == AF.AffiliationId;
+   ```
+
+   Press the **SHIFT + ENTER** keys to run the code in this block. You see following output:
+
+   ```
+   OK
+   ```
+
+## Query and visualize result 
+
+In this section, you query top authors by h-index and visualize the result.
+
+1. Query top authors with highest h-index. Paste the following code in a new cell. Press the **SHIFT + ENTER** keys to run the code in this block.
+
+   ```sql
+   %sql
+   -- Filter authors with top hindex
+   SELECT
+       DisplayName,
+       AffiliationDisplayName,
+       PaperCount,
+       TotalEstimatedCitation,
+       HIndex
+   FROM AuthorHIndex 
+   ORDER BY HIndex DESC, AuthorId
+   LIMIT 100;
+   ```
+
+1. Select the **table** icon to see result in table form.
+
+   ![Author H-Index table](media/databricks/hindex-table.png "Verify sample table")
+
+1. Select the **graph** icon to see result in graph form.
+
+   ![Author H-Index graph](media/databricks/hindex-graph.png "Verify sample table")
+
+## Clean up resources
+
+After you finish the tutorial, you can terminate the cluster. From the Azure Databricks workspace, select **Clusters** on the left. For the cluster to terminate, under **Actions**, point to the ellipsis (...) and select the **Terminate** icon.
+
+![Stop a Databricks cluster](media/databricks/terminate-databricks-cluster.png "Stop a Databricks cluster")
+
+If you don't manually terminate the cluster, it automatically stops, provided you selected the **Terminate after \_\_ minutes of inactivity** check box when you created the cluster. In such a case, the cluster automatically stops if it's been inactive for the specified time.
+
```

Comment:
Should we also add an optional step for deleting the cluster?

Reply:
I think that might be too much. ;)